acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/ExternalMetricPublisherAcceptanceTest.java

```diff
@@ -31,7 +31,7 @@ void shouldPublishDataFromPrometheus() throws Throwable {
             config ->
                 config
                     .withExternalMetricsClient(externalMetricNode, 1)
-                    .withInteropNumberOfValidators(VALIDATOR_COUNT));
+                    .withInteropValidators(0, VALIDATOR_COUNT));
```

Comment:
looking at the actual metrics it's expecting it looks like it actually publishes two separate metrics: `validator_total` and `validator_active`.  Previously they've always been (incorrectly) the same value.  So I think we actually want to go back to the validator setup this had previously but probably be more explicit.  eg:

```
.withInteropNumberOfValidators(VALIDATOR_COUNT)
.withInteropValidators(0, 64)
```

And then I'd probably rename `VALIDATOR_COUNT` to `ACTIVE_VALIDATOR_COUNT` and make the 64 a constant `TOTAL_VALIDATOR_COUNT`.  We'll then need to update the line:
```
    externalMetricNode.waitForValidatorMetricPublication(VALIDATOR_COUNT);
```

to take two params, one for the expected total validator count and one for the expected active validator count.  

And I think the production code is now doing the right thing because of your changes and should report the active and total values accurately.

Reply:
I can work on this in a few weeks, but I don't know if I understood correctly the first part of the request.

As I understood, `.withInteropValidators` sets the number of validators keys owned by the node under testing, while `.withInteropNumberOfValidators` sets the total number of validators in the entire network, but reading your comment it seems to be the contrary.

I my first interpretation is correct, the final code should look like this:

```java
.withInteropNumberOfValidators(TOTAL_VALIDATOR_COUNT)
.withInteropValidators(0, ACTIVE_VALIDATOR_COUNT)
```

Is it right?

Reply:
Yes your interpretation looks right to me - sorry I think I got things backwards when talking about the renaming.

Reply:
Hi @ajsutton, I finally fixed the PR. Tests pass; it should be all fine now.